cmd/ctr/commands/version/version.go

```diff
@@ -28,13 +28,13 @@ var Command = cli.Command{
 			return err
 		}
 		fmt.Println("Server:")
-		fmt.Printf("  Version: %s\n", v.Version)
-		fmt.Printf("  Revision: %s\n", v.Revision)
+		fmt.Println("  Version: ", v.Version)
+		fmt.Println("  Revision:", v.Revision)
 		if v.Version != version.Version {
-			fmt.Fprintf(os.Stderr, "WARNING: version mismatch\n")
+			fmt.Println(os.Stderr, "WARNING: version mismatch")
```

Comment:
`Fprintln` same for the next changed lline

Reply:
Doh! Thanks